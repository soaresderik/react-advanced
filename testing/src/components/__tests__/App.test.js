import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import App from "../App";
import CommentBox from "../CommentBox";
import CommentList from "../CommentList";

let wrapped;

beforeEach(() => {
  wrapped = shallow(<App />);
});

it("shows a comment box", () => {
  expect(wrapped.find(CommentBox).length).toEqual(1);
});

it("show a comment list", () => {
  expect(wrapped.find(CommentList).length).toEqual(1);
});

it("render <App /> correctly", () => {
  expect(toJson(wrapped)).toMatchSnapshot();
});
