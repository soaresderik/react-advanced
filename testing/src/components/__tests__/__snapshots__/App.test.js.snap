// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <App /> correctly 1`] = `
<div>
  <Connect(CommentBox) />
  <Connect(CommentList) />
</div>
`;
